tinker-actions/src/kernelupgrd/kernel_upgrade.sh

```diff
@@ -50,7 +50,16 @@ mount --bind /etc/resolv.conf /mnt/etc/resolv.conf
 
 mv /mnt/etc/apt/apt.conf.d/99needrestart /mnt/etc/apt/apt.conf.d/99needrestart.bkp 
 
-KERNEL_VERSION=linux-image-6.8.0-52-generic
+KERNEL_VERSION_2204=linux-image-6.8.0-52-generic
+KERNEL_VERSION_2404=linux-image-6.11.0-9-generic
+
+# Detect Ubuntu version and set KERNEL_VERSION accordingly
+os_release_file="/mnt/etc/os-release"
```

Comment:
is this indep of the baseline version? does this path apply to 22.04 and 24.04 ?

Reply:
yes independent of the baseline version & yes the path is found on both.